.vale/styles/style_guide/Repetition.yml

```
@@ -0,0 +1,10 @@
+---
+# Error: style_guide.Repetition.yml
+
+# catch any instances of repeated words. E.g., "This is is a sentence"
+extends: repetition
+message: "'%s' is repeated."
+level: error
+alpha: true
+tokens:
+  - '[^\s]+'
```

Figured out what the problem was. When you press return while making a suggestion, GitHub helpfully inserts a tab to match the indentation level. So Gui's suggestion added some whitespace instead of a fully blank line

```suggestion
  - '[^\s]+'

```